src/components/DocsNavigation/index.tsx

```diff
@@ -143,11 +143,11 @@ export const DocsNavigation = ({
             </div>
           )}
           <Accordion.Root
-            defaultValue={[...openTopicPreferences, currentTopic]}
+            defaultValue={currentTopic}
             onValueChange={(value) =>
               window.localStorage.setItem(openTopicsLocalStorageKey, JSON.stringify(value))
             }
-            type="multiple"
+            type="single"
```

Comment:
This now doesn't allow all sections to be closed. If one is open, it can't be closed.
Additionally, onChange needs to be modified so that localStorage clears up and stores only one section as expanded. Otherwise, when you refresh the page, the expanded section isn't the last one you expanded.